tests/acceptance/blockchain/conditions/conftest.py

```diff
@@ -181,7 +185,9 @@ def subscription_manager_is_active_policy_condition(test_registry, agency):
     )
     condition = ContractCondition(
         contract_address=subscription_manager.contract.address,
-        function_abi=[subscription_manager.contract.find_functions_by_name('isPolicyActive')[0].abi],
+        function_abi=subscription_manager.contract.find_functions_by_name(
+            "isPolicyActive"
+        )[0].abi,
```

Comment:
Can be simplified by using `get_function_by_name` (see PR https://github.com/theref/nucypher/pull/1).